lib/rucio/api/replica.py

```diff
@@ -212,3 +212,13 @@ def list_datasets_per_rse(rse, filters=None, limit=None):
     :returns: A list of dict dataset replicas
     """
     return replica.list_datasets_per_rse(rse, filters=filters, limit=limit)
+
+
+def get_suspicious_files(rse_expression, younger_than=None, nattempts=None):
+    """
+    List the list of suspicious files on a list of RSEs
+    :param rse_expression: The RSE expression where the suspicious files are located
+    :param younger_than: datetime object to select the suspicious replicas younger than this date.
+    :param nattempts: The number of time the replicas have been declared suspicious
+    """
+    return replica.get_suspicious_files(rse_expression=rse_expression, younger_than=younger_than, nattempts=nattempts)
```

Comment:
in the other API calls we usually have seconds and not datetime objects to pass around. any particular reason why datetime objects are more fitting here? something like younger_than=1234 seconds is not enough?

Reply:
The API is only used via the WebUI, and I found it more convenient to use datetime